lib/t_web/channels/profile_channel.ex

```diff
@@ -71,6 +71,13 @@ defmodule TWeb.ProfileChannel do
     {:reply, reply, socket}
   end
 
+  # TODO refresh after one hour
```

Comment:
why?

Reply:
token expires in 3600 based on the reply to this request

Reply:
check how apns token works in apns/token.ex